src/types/h5p/index.d.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * The H5P API reference
+ * https://h5p.org/documentation/api/H5P.html
+ *
+ * We only specify used members, so feel free to add properties according to the above spec
+ */
+export namespace H5P {
+  /** Async error handling */
+  type ErrorCallback = (error: unknown) => void;
+  /** Callback type for event listeners */
+  type EventCallback = (event: Event) => void;
+
+  /**  The Event class for the EventDispatcher.  */
+  class Event {
+    constructor(
+      type: string,
+      data: unknown,
+      extras?: { bubbles?: boolean; external?: boolean },
+    );
+    /**
+     * Get bubbling status
+     * @returns true if bubbling false otherwise
+     */
+    getBubbles(): boolean;
+    /**
+     * Prevent this event from bubbling up to parent
+     */
+    preventBubbling();
+    /**
+     * Try to schedule an event for externalDispatcher
+     * @returns true if external and not already scheduled, otherwise false
+     */
+    scheduleForExternal(): boolean;
+  }
+
+  /** Used for xAPI events */
+  class XAPIEvent extends Event {
+    /** List of verbs defined at ADL xAPI Vocabulary */
+    static allowedXAPIVerbs: Array<string>;
+    /** Get content xAPI ID */
+    getContentXAPIId(instance: H5P.Instance);
+    /** Get the max value of the result - score part of the statement */
+    getMaxScore(): number | null;
+    /** Get the raw valu of the result - score part of the statement */
+    getScore(): number | null;
```

Comment:
What are these scores?

Reply:
They come from the H5P API reference at https://h5p.org/documentation/api/H5P.html
I've only added the basic types from the global H5P namespaces for Typescript.